.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,29 @@
+---
+name: Bug report
+about: Report an issue with music21
+title: ''
+labels: ''
+assignees: ''
+
+---
+
```

Comment:
asking for music21 version is probably important too -- we still get bug reports for 1.0 sometimes... and definitely for 5.7.